proto/cheqd/did/v2/query.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package cheqd.did.v2;
+
+import "google/api/annotations.proto";
+import "cheqd/did/v2/diddoc.proto";
+
+option go_package = "github.com/cheqd/cheqd-node/x/cheqd/types/v2";
+
+service Query {
+	rpc DidDoc(QueryGetDidDocRequest) returns (QueryGetDidDocResponse) {
+		option (google.api.http).get = "/cheqd/v1/diddoc/{id}";
```

Comment:
Seems incorrect. Should be `cheqd/did/v2/diddoc` right?

Reply:
Yeah... I left paths for future because they are not breaking changes either (+ nobody uses HTTP gateway).